source/user_guide.md

```diff
@@ -843,7 +851,8 @@ documentation for a more detailed description of the commands that follow.
 
  - `recall /app`: Given a placed application graph instance (or multiple),
    informs all Motherships that host the application to recall it (forget about
-   it completely), unless it is running (it will need to be stopped first).
+   it completely), unless it is running (it will need to be stopped first), or
+   unless it has been initialised (it will need to be started, then stopped).
```

Comment:
We could change this behaviour to allow things at the barrier to respond to KILL packets?

Reply:
I reckon so

Reply:
Created #262 so we can discuss further.